Core/UI/steering/steering.c

```diff
@@ -614,3 +614,177 @@ void handle_ptt_message(void) {
     ecu_ack = true;
   }
 }
+
+void set_label_color_hv_feedbacks(int label, int i) {
```

Comment:
Also make sure that these labels will update only when the this tab (tab errors) is active. You can update the labels of all the errors when you are in this tab. Please use the `CHECK_CURRENT_TAB` macro as it is used in tab sensors and in tab racing.